code_generator/GdkCodeGenerator/Templates/UnityGameObjectComponentDispatcherGenerator.tt

```diff
@@ -83,18 +83,20 @@ namespace <#= qualifiedNamespace #>
             public override void InvokeOnEventUserCallbacks(GameObjectDispatcherSystem gameObjectDispatcherSystem)
             {
 <# for (var j = 0; j < eventDetailsList.Count; j++) { #>
-                var entities = EventsReceivedComponentGroups[<#= j #>].GetEntityArray();
```

Comment:
This wasn't compiling if you had more than 1 event so I put the parts within `{}`

Reply:
For an easier review: 
![image](https://user-images.githubusercontent.com/31209570/43732274-19236f04-99a9-11e8-92c0-264a91937a16.png)